package.json

```diff
@@ -7,7 +7,7 @@
     "test": "node tests/test_template_generator.js"
   },
   "dependencies": {
-    "assemblyscript": "lrettig/assemblyscript#ewasm"
+    "assemblyscript": "github:AssemblyScript/assemblyscript"
```

Comment:
It's good to switch back, good call. We should keep an eye if they merge that branch, then we can remove the hardcoded `49` or whatever that we use to get from the import

Reply:
We now have our own fork which exports NodeKind